src/Units/UnitModel.java

```diff
@@ -60,32 +59,29 @@ public static String standardizeID(String input)
     return input.toLowerCase().replaceAll(" ", "_").replaceAll("-", "_");
   }
 
+  public int costBase;
+  public int baseMovePower;
+  public MoveType baseMoveType;
+  public ArrayList<UnitActionFactory> baseActions = new ArrayList<UnitActionFactory>();
+  public int baseCargoCapacity = 0;
+
+  // If you want to modify anything below this line, you're gonna have to implement it into UnitContext
```

Comment:
`healableHabs` is below the line, but it's modified for BattleMech (and not in UnitContext).

Reply:
BattleMech doesn't count because it's a *new* unit type, not really a CO changing an existing unit type.
I suppose I should be more precise with the comment, though.

Reply:
I could say the same for max HP, except Tech doesn't actually change the field, she just kinda ignores it.

Reply:
I would kinda prefer she actually modify the max HP so we can iron out those semantics, but what we have today works and we don't have any other planned COs that use the mechanic... so it's not a super high priority.